miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSubprojectService.java

```diff
@@ -26,7 +26,7 @@
   protected static final Logger log = LoggerFactory.getLogger(DefaultSubprojectService.class);
 
   @Autowired
-  private SQLProjectDAO sqlProjectDAO;
+  private HibernateProjectDao sqlProjectDAO;
```

Comment:
There's no bean definition with the name "sqlProjectDao." There's one called "projectStore" in db-config.xml, but the name shouldn't matter at all now, as that should be the only ProjectStore bean. That bean still has `class="uk.ac.bbsrc.tgac.miso.sqlstore.SQLProjectDAO"`, which does need updated.

Autowired type should be the interface rather than the implementation as well.